announcements/2021/week05.md

```diff
@@ -0,0 +1,20 @@
+# :zap: This week in provider-independent announcements (February 05, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[External DNS v2.0.0](https://docs.giantswarm.io/changes/managed-apps/external-dns-app/v2.0.0/) allows customers to install and use External DNS for their own purpose, separate from the preinstalled app in AWS and Azure clusters. [External DNS v2.1.0](waiting for link) adds guardrails (dry-run mode, etc.) to help prevent misconfigurations. :warning: [Requires NGINX IC v1.14.0+](https://github.com/giantswarm/external-dns-app#limitations).
```

Comment:
@glitchcrab announcement text for new release. please check if makes sense 😀

Reply:
and maybe add detail if it's too vague..

Reply:
no i think that makes sense, although i'd prefer customers to just jump straight to 2.1.0 because it's much safer for them to use